pandas/tests/test_window.py

```diff
@@ -4749,3 +4782,20 @@ def test_rolling_cov_offset(self):
 
         expected2 = ss.rolling(3, min_periods=1).cov()
         tm.assert_series_equal(result, expected2)
+
+
+class TestCustomIndexer:
```

Comment:
This class isn't necessary. Since we have `pytest` there's really no reason not to use top-level-function-style tests.